cradle-core/src/test/java/com/exactpro/cradle/messages/GroupedMessageBatchToStoreJoinTest.java

```diff
@@ -42,7 +44,7 @@ public void testJoinEmptyBatchWithOther() throws CradleStorageException, Seriali
         assertTrue(emptyBatch.addBatch(batch));
 
         assertEquals(emptyBatch.getMessageCount(), 5);
-        assertEquals(emptyBatch.getBatchSize(), getBatchSize(emptyBatch));
+        assertEquals(emptyBatch.getBatchSize(), emptyBatch.getBatchSize());
```

Comment:
What is the point of that check? Probably, we should check that the empty batch now has the same size as the batch we added
```suggestion
        assertEquals(batch.getBatchSize(), emptyBatch.getBatchSize());
```